app/models/like.rb

```diff
@@ -1,4 +1,10 @@
 class Like < ApplicationRecord
-  belongs_to :author
+  belongs_to :author, class_name: 'User'
   belongs_to :post
+
+  after_save :update_likes_counter
+
+  def update_likes_counter
+    post.increment!(:comments_counter)
```

Comment:
- [ ] kindly consider making sure this method updates  the likes counter 😀  for a post as we are in `likes` counter not `comments_counter ` doing so will make this model meet the project requirements

Reply:
Done :+1: